stdlib/Sockets/src/addrinfo.jl

```diff
@@ -267,7 +269,7 @@ julia> getipaddrs(IPv6)
  ip"fe80::445e:5fff:fe5d:5500"
 ```
 """
-function getipaddrs(addr_type::Type{T}, include_lo::Bool=false) where T<:IPAddr
+function getipaddrs(addr_type::Type{T}=IPAddr; loopback::Bool=false) where T<:IPAddr
```

Comment:
Thanks for adding in the 0-argument method, seems like I forgot that in my PR. :heart:

Reply:
You had it, it just couldn’t be specified like this because of the interaction with non-keyword loopback argument.